.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Can you add a comment with link to the documentation of this file?

Also consider add an inline comment to explain what this config value is expected to do?